scheduler/requirements.txt

```diff
@@ -1,6 +1,8 @@
 apache-airflow-providers-postgres==5.10.0
 apache-airflow-providers-redis==3.5.0
 apache-airflow-providers-celery==3.5.1
+apache-airflow[statsd]
```

Comment:
Can you please pin these new dependencies?

Reply:
Done